grpc/dataplane_grpc.proto

```diff
@@ -0,0 +1,90 @@
+syntax = "proto3";
+
+package config;
+
+option go_package = "dataplane/grpc";
+
+service ConfigService {
+  rpc GetConfig (GetConfigRequest) returns (GlobalConfig);
+  rpc GetConfigGeneration (GetConfigGenerationRequest) returns (GetConfigGenerationResponse);
+  rpc UpdateConfig (UpdateConfigRequest) returns (UpdateConfigResponse);
+}
+
+message GetConfigRequest {}
+
+message UpdateConfigRequest {
+  GlobalConfig config = 1;
+}
+
+message UpdateConfigResponse {
+  Error error = 1;
+  string message = 2;
+}
+
+enum Error {
+  ERROR_NONE = 0;
+  ERROR_VALIDATION_FAILED = 1;
+  ERROR_APPLY_FAILED = 2;
+  ERROR_UNKNOWN_ERROR = 3;
+}
+
+enum IfType {
```

Comment:
I'm not sure I understand the semantics of this field.
My impression is that it is mixing the purpose of an interface with its type?
Shouldn't those go separate? IMO, uplink and management sound more like an interface description?

Reply:
Also, are we going to model Vxlan interfaces?

Reply:
I guess with given fact that Dataplane will be responsible for entire config, the model should be changed a lot